applications/serial_lte_modem/src/lwm2m_carrier/slm_at_carrier.c

```diff
@@ -16,6 +16,8 @@ LOG_MODULE_REGISTER(slm_carrier, CONFIG_SLM_LOG_LEVEL);
 
 /**@brief LwM2M Carrier operations. */
 enum slm_carrier_operation {
+	/* Carrier Connect Operation */
+	CARRIER_OP_REQUEST_AUTO_CONNECT,
```

Comment:
The `REQUEST` prefix corresponds to the `lwm2m_carrier_request()` API in lwm2m_carrier.h. This operation is not a Carrier library API, so I'd name it simply `CARRIER_OP_AUTO_CONNECT`, and then improve the comment here since there is no documentation for it as opposed to the library's APIs, and it is not intuitive to think that this will only work for NIDD as is now.

Reply:
It's a leftover or copy/paste CARRIER_OP_REQUEST_LINK_UP :)